File: Software Technologies/18_Exam/JavaScript/models/issue.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    let Issue = sequelize.define('Issue', {
        title:{
            type: Sequelize.TEXT,
            allowNull: false,
        },
        content:{
            type: Sequelize.TEXT,
            allowNull: false,
        },
        priority:{
            type: Sequelize.INTEGER,
            allowNull: false,
        }
    },{
        timestamps: false,
    });

    return Issue;
};

File: Software Technologies/18_Exam/C#/IssueTracker/Models/Issue.cs
namespace IssueTracker.Models
{
    using System.ComponentModel.DataAnnotations;

    public class Issue
    {
        [Key]
        public int Id { get; set; }

        public string Title { get; set; }

        public string Content { get; set; }

        public int Priority { get; set; }
    }
}
